LibrarySets/CMS/Cql/AHAOverall.cql

```diff
@@ -139,27 +139,27 @@ define fluent function overlapsAfterHeartFailureOutpatientEncounter(Event Choice
             where
               case
                 when Event is Condition then
-                  (Event as Condition).prevalenceInterval() overlaps after day of ModerateOrSevereLVSDHFOutpatientEncounter.period
-                    and (Event as Condition).isConfirmedActiveDiagnosis()
+                  Event.prevalenceInterval() overlaps after day of ModerateOrSevereLVSDHFOutpatientEncounter.period
```

Comment:
This overwrites my manual changes here of course - since we don't do choice types nor keep track of the types determined by a type check, we cannot handle these expressions without adding manual casts.